### PR TITLE
version bump LOK to 0.5.14

### PR DESCRIPTION
**Pull request by gbirman**

### CODE REVIEW

**Comment by whutchinson98** [comment]
Can you put this inside of your other PR so once we merge it in builds auto kick off. I wouldn't want to accidentally merge this in first and have a build not contain your other changes

**Comment by gbirman** [comment]
Discussed with @synoet and it seems like the canonical thing to do is to separate version bumps from changes. Adding the `do not merge` label to prevent merging until the outstanding PR is in main.

**Review by whutchinson98 — APPROVED** [review]

**Pull request merged**
